docker/vast/docker-compose.yaml

```diff
@@ -6,6 +6,7 @@ services:
     # at the earliest, which will be the first release to offer a stable tag for
     # Docker Hub.
     image: ${VAST_CONTAINER_REGISTRY:-docker.io}/tenzir/vast:${VAST_CONTAINER_REF:-latest}
+    container_name: vast-server
```

Comment:
@dominiklohmann we need this for Cortex to be able to dynamically create a network link between the containerized jobs it creates and VAST (`CORTEX_NETWORK_MODE=container:vast-server`). Is this tiny addition in the vast service fine for you?